src/config.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  clientsUrl: "http://localhost:3000/clients",
```

Comment:
In config should be only 1 url
```
apiUrl = "http://localhost:3000"
```
and inside stores you can code at the begging of file
```
const baseUrl = `${config.apiUrl}/clients`
const baseUrl = `${config.apiUrl}/appointments`
```

Reply:
done